OSDOCS-9826: adds Y+2 update paths MicroShift (#73988)
openshift/openshift-docs: microshift_updating/microshift-update-options.adoc

[IMPORTANT]
====

> Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step. Updating 4.14 to 4.15 first is not required.

Reviewer: ```suggestion
Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step without updating to 4.15 first .
```

====

[id="microshift-update-options-standalone-updates_{context}"]
== Standalone {microshift-short} updates

> You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required to update {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.

Reviewer: ```suggestion
You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required before updating {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.
```

Reviewer: Under what conditions can update without reinstalling apps or operators. The second sentence doesn't seem to answer this question. (Or, maybe it does and I am ignorant here.).

Author: Separate thoughts, changed the structure a bit. In reality, we can'r answer that question here, customer has to figure that out based on custom apps and whatnot.
